homeassistant/components/tessie/strings.json

```diff
@@ -35,7 +35,12 @@
         }
       },
       "route": {
-        "name": "Route"
+        "name": "Route",
+        "state_attributes": {
+          "destination": {
+            "name": "Destination"
+          }
+        }
```

Comment:
Why do we have this here?

Reply:
Because I forgot to remove it, will do so immediately